<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupDialog</class>
 <widget class="QDialog" name="SetupDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>560</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Setup</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QTabWidget" name="setupTabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>670</width>
     <height>538</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="documentsTab">
    <attribute name="title">
     <string>Docume&amp;nts</string>
    </attribute>
    <widget class="QPushButton" name="startSellerSetup">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Seller Invoice</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startBuyerSetup">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>150</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Buyer Invoice</string>
     </property>
    </widget>
    <widget class="QPushButton" name="checkSetupButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>150</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Checks</string>
     </property>
    </widget>
    <widget class="QPushButton" name="marketReportSetupButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>150</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>M&amp;arket Report</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reportSetupButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>150</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Reports</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="codesTab">
    <attribute name="title">
     <string>Codes</string>
    </attribute>
    <widget class="QPushButton" name="commEditButton">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>30</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Commissions</string>
     </property>
    </widget>
    <widget class="QPushButton" name="chargeEditButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Charges</string>
     </property>
    </widget>
    <widget class="QPushButton" name="codeEditButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Codes</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="systemTab">
    <attribute name="title">
     <string>S&amp;ystem</string>
    </attribute>
    <widget class="QPushButton" name="printerSetupButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>150</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Printers</string>
     </property>
    </widget>
    <widget class="QPushButton" name="externSetupButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>150</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Clerk / Penner</string>
     </property>
    </widget>
    <widget class="QPushButton" name="setupScaleButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>150</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
    <widget class="QPushButton" name="setupDisplaysButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>150</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Displays</string>
     </property>
    </widget>
    <widget class="QPushButton" name="setupVideoCameraButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>150</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Video camera</string>
     </property>
    </widget>
    <widget class="QPushButton" name="setupFaxModemButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>150</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Fax modem</string>
     </property>
    </widget>
    <widget class="QPushButton" name="setupSaleManagerButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>150</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Sale Manager</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="marketSetupTab">
    <attribute name="title">
     <string>Mar&amp;ket Setup</string>
    </attribute>
    <widget class="QLabel" name="textLabel3_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Address</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="textLabel3_2_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>City</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="textLabel3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="marketZipEdit">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>100</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="textLabel3_2_2_2_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>100</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Zip</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="marketStateEdit">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>100</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="textLabel3_2_2_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>100</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>State</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="marketCityEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>150</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="marketNameEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>410</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="marketAddressEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>410</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="saveMarketButton">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>450</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="textLabel3_2_2_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>ID #:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="marketIdEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>140</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="rfPortEdit">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>410</y>
       <width>170</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="transientBuyerBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>190</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QLabel" name="textLabel3_2_2_3_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>230</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>First Transient Buyer:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="textLabel1">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>410</y>
       <width>160</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>RF Tag Port:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="rfTagOnCheckBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>410</y>
       <width>160</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Use RF Tags</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="selectiveTagReportCheck">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>330</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Individual Se&amp;xes For Tag Report</string>
     </property>
     <property name="shortcut">
      <string>Alt+X</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="countryOfOriginCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>330</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Use Country Of Origin Tracking</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="dailyFMPhoneMatch">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>380</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Use phone num&amp;ber as seller number.</string>
     </property>
     <property name="shortcut">
      <string>Alt+B</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="dailyFMCheckinTickets">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>355</y>
       <width>380</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Print checkin tickets from Daily FM.</string>
     </property>
     <property name="shortcut">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="webAuctionsTab">
    <attribute name="title">
     <string>&amp;Web Auctions</string>
    </attribute>
    <widget class="QWidget" name="layout6">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>16</y>
       <width>571</width>
       <height>480</height>
      </rect>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QCheckBox" name="enableCattleUSACheckBox">
        <property name="text">
         <string>Enable Cattle USA Integration</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="cattleUSAIdTextLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Cattle USA ID</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="cattleUSAIdLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="textLabel1_2">
          <property name="text">
           <string>Server</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="cattleUSAServerEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="spacer2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>470</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="enableDVAuctionCheckBox">
        <property name="text">
         <string>Enable DVAuction Integration</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="dvAuctionIdTextLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>DVAuction ID</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dvAuctionIdLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="textLabel1_4">
          <property name="text">
           <string>Server</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dvAuctionServerEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="spacer2_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>470</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <spacer name="spacer1">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>161</width>
            <height>31</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="saveWebAuctionsButton">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>setupTabWidget</tabstop>
  <tabstop>startSellerSetup</tabstop>
  <tabstop>startBuyerSetup</tabstop>
  <tabstop>checkSetupButton</tabstop>
  <tabstop>marketReportSetupButton</tabstop>
  <tabstop>reportSetupButton</tabstop>
  <tabstop>commEditButton</tabstop>
  <tabstop>chargeEditButton</tabstop>
  <tabstop>codeEditButton</tabstop>
  <tabstop>printerSetupButton</tabstop>
  <tabstop>externSetupButton</tabstop>
  <tabstop>setupScaleButton</tabstop>
  <tabstop>setupDisplaysButton</tabstop>
  <tabstop>marketNameEdit</tabstop>
  <tabstop>marketAddressEdit</tabstop>
  <tabstop>marketCityEdit</tabstop>
  <tabstop>marketStateEdit</tabstop>
  <tabstop>marketZipEdit</tabstop>
  <tabstop>marketIdEdit</tabstop>
  <tabstop>transientBuyerBox</tabstop>
  <tabstop>rfTagOnCheckBox</tabstop>
  <tabstop>rfPortEdit</tabstop>
  <tabstop>saveMarketButton</tabstop>
 </tabstops>
 <includes>
  <include location="local">./dao/animalcodedao.h</include>
  <include location="local">./types/animalcode.h</include>
  <include location="global">vector</include>
  <include location="local">dao/commissiondao.h</include>
  <include location="local">dao/setupdao.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
